configs/projects/puppet-agent.rb

```diff
@@ -92,6 +92,7 @@
   proj.component "puppet"
   proj.component "facter"
   proj.component "hiera"
+  proj.component "leatherman"
   proj.component "marionette-collective"
   proj.component "cpp-pcp-client"
   proj.component "pxp-agent"
```

Comment:
Isn't leatherman a dependency on cpp-cpc-client and pxp-agent as well? If it is, we should include it as a build_requires in those config files just for the sake of clarity